src/getTargetPath.js

```diff
@@ -0,0 +1,57 @@
+const getPath = (steps) => {
+  let tag, stringSteps = [];
+  for (let i = 0, element = steps[i]; element; element = element.parentNode, i++) {
+    if (element === document || element === window) break;
+
+    tag = element.tagName.toLowerCase();
+
+    if (tag === 'html') break;
+
+    // cut path when id exists
+    if (element.id) {
+      tag = `#${element.id}`;
+      stringSteps.push(tag);
+      break;
+    }
+
+    // if parent has multiple same tags so we should defined which child it is 
+    if (Array.prototype.slice.call(element.parentNode.children).filter(child => child.tagName.toLowerCase() === tag).length > 1) {
+      const allSiblings = Array.prototype.slice.call(element.parentNode.children);
+      const index = allSiblings.indexOf(element) + 1;
+      if (index > 0) {
+          tag += `:nth-child(${index})`;
+      }
+    }
+
+    // exclude SVGAnimatedString className objects
+    if (typeof element.className === 'string' && element.className) {
+      const classes = element.className.replaceAll(' ', '.');
+      tag += `.${classes}`;
+    }
+
+    stringSteps.push(tag);
+  }
+
+  return stringSteps.reverse().join('>');
+}
+
+export default (e) => {
+  let path = [];
+
+  // handling missing method in IE11 / Edge
+  if (!e.composedPath) {
+    const target = e.target;
+
+    while (target.parentNode !== null) {
+      path.push(e.target);
+      target = target.parentNode;
+    }
+    path.push(document);
+
+    path = getPath(path);
+  }
```

Comment:
add else statement here with `path = e.composedPath()` and then `return getPath(path)`